apimonitor/monitor/health.py

```
@@ -0,0 +1,162 @@
+                        'url': url,
+                        'status': status_code,
+                        'ok': ok,
+                        'latency_ms': latency_ms,
+                        'attempt': attempt + 1,
+                        'ts': time.time()
+                    }
+                )
+                
+                return result
+                
+        except TimeoutException as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)